rules/windows/registry_event/sysmon_asep_reg_keys_modification.yml

```diff
@@ -4,35 +4,181 @@ description: Detects modification of autostart extensibility point (ASEP) in reg
 status: experimental
 references:
     - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1060/T1060.yaml
+    - https://attack.mitre.org/techniques/T1547/001/
 tags:
     - attack.persistence
-    - attack.t1060 # an old one
     - attack.t1547.001
-date: 2019/10/21
-modified: 2020/09/06
-author: Victor Sergeev, Daniil Yugoslavskiy, oscd.community
+    - attack.t1060      # an old one
+date: 2019/10/25
+modified: 2020/10/13
+author: Victor Sergeev, Daniil Yugoslavskiy, Gleb Sukhodolskiy, Timur Zinniatullin oscd.community
 logsource:
     category: registry_event
-    product: windows
+    product: windows 
+level: medium
 detection:
-    selection:
-        TargetObject|contains:
-            - '\software\Microsoft\Windows\CurrentVersion\Run'
-            - '\software\Microsoft\Windows\CurrentVersion\RunOnce'
-            - '\software\Microsoft\Windows\CurrentVersion\RunOnceEx'
-            - '\software\Microsoft\Windows\CurrentVersion\RunServices'
```

Comment:
Gleb, I think we can follow the same logic with RunOnce and RunOnceEx.

For example:
\SOFTWARE\Microsoft\Windows\CurrentVersion\Run

Would match:
\SOFTWARE\Microsoft\Windows\CurrentVersion\RunOnceEx
\SOFTWARE\Microsoft\Windows\CurrentVersion\RunOnce
\SOFTWARE\Microsoft\Windows\CurrentVersion\Run
\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Terminal Server\Install\Software\Microsoft\Windows\CurrentVersion\RunonceEx
\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Terminal Server\Install\Software\Microsoft\Windows\CurrentVersion\Runonce
\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Terminal Server\Install\Software\Microsoft\Windows\CurrentVersion\Run

Plus, the ones you mentioned. What do you guys think?

Reply:
Thanks! I missed that. Done 👍